### DISCUSSION TITLE
Amigos como estan? alguno que me revise el repo a ver que tal me va con la consigna. Grax

### DISCUSSION THREAD

**Discussion by ghost** [Launch-X-Latam/MisionFrontEnd#123] [Q&A] [answered]
Resulta que para la primera practica decidi hacer un mero analisis de requisitos y comence con el sketch de como seria la web y su comportamiento, antes de seguir me gustaria que alguien lo chequeara a ver que le parece.
Muchas gracias explorers!xD

**Comment by Amhedriel** [comment]
Buenas,

Se ve bastante bien.

**Reply by ghost** [reply]
gracias compa

**Comment by EsthebanDiego** [comment]
Tambien voy empezando pero se ve bien

**Reply by ghost** [reply]
gracias

**Comment by MartiRF** [comment, marked as answer]
Puedes agregar un archivo readme.md para mostrar tus imagenes en el inicio. hay varias guías para usar el markdown.

**Reply by ghost** [reply]
ah investigare eso, muchas gracias

**Comment by AnaPatino18** [comment]
Pues van muy bien, yo apenas saliendo del shock para comprender el requerimento, pero ya iniciando

**Reply by ghost** [reply]
thanks

**Comment by JuanSposada** [comment]
Va muy bien compañero. Me diste una buena guía de como llevarlo creo que vamos por buen camino

**Comment by ErickOrtiz0298** [comment]
Se ve bien n.n igual los invito revisar la mía. Saludos

**Comment by lODIN007l** [comment]
va genial la verdad
